docs/avocado/blue/examples/glossed_text_story.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<burritoMetadata id="dcs::0123456789abcdef" revision="1" version="0.1">
+<burritoMetadata id="urn:dcs::unfoldingword/en_obs" revision="94263a75308f91567c3c77ba862d53cbf6c80378" version="5">
```

Comment:
@version here is the schema version, so it should be 0.1. If this is confusing (which I guess it is given the other fields in the root element), maybe we need a better word than "version". "schemaVersion" would be ugly but very hard to misunderstand.

Reply:
I do think something like schemaVersion or sbVersion would be clearer.